packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -4223,6 +4214,8 @@ class InputDecorationTheme with Diagnosticable {
     filled,
     Object.hash(
       fillColor,
+      activeIndicatorBorder,
```

Comment:
It looks like `inputStyle` is missing here and other places below. Please verify that all three added fields are accounted for in each of the boilerplate methods.

Reply:
Done